scripts/deployment/main.py

```diff
@@ -28,7 +28,10 @@ def load_contracts_info(contracts_path: Path) -> dict[str, tuple]:
     for path in contracts_path.glob("*.json"):
         with path.open() as fp:
             info = json.load(fp)
-        contracts[info["contractName"]] = (info["abi"], info["bytecode"])
+        try:
+            contracts[info["contractName"]] = (info["abi"], info["runtimeBytecode"]["bytecode"])
+        except KeyError:
+            pass
```

Comment:
Why this `try...except` block?

Reply:
abstract contracts are coming in the built path also that they dont have any bytecode basically.

Reply:
gone